### PR TITLE
[CHI-346] Added newer version availability check + made version dropdown dynamic

### PR DESCRIPTION
**Pull request by janibolkvadze**
@dani-cl-madrid @SergioQCL @jllr @mattnickles 

https://ctl.atlassian.net/browse/CHI-346

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/290/ for more details.

File: src/website/assets/scripts/chi-docs.js [hunk 1]
```diff
@@ -174,6 +174,57 @@ onLoad(() => {
...
+    }
+  }
+
+  // checkChiCurrentVersion("1.2.3");
+  function checkChiCurrentVersion(currentVersion) {
+      if(window.chiCurrentVersion.toString() !== currentVersion.toString()) {
+        var chiVersionCheckSelector = document.getElementById('chiVersionCheck');
+
+        chiVersionCheckSelector.innerHTML = `
+        <div class="m-alert -bubble -info" role="alert">
```
Comment (mattnickles) [inline src/website/assets/scripts/chi-docs.js]: Let's render this as a banner: `m-alert -banner -center -info` and place it above `o-header__content` with a 100% width `-w--100` so it stretches the entire width of the screen for maximum visibility. We'll need to apply a class on body or docs-container if the version isn't current and then use that class to adjust the padding-top of `docs-body` to `6.5rem` and `docs-body__aside` top value to `104px` to make room for the banner.

image

File: src/website/assets/scripts/chi-docs.js [hunk 2]
```diff
@@ -174,6 +174,57 @@ onLoad(() => {
...
+          <i class="a-icon -sm--2 icon-circle-info -text--info"></i>
+          <a href="https://assets.ctl.io/chi/">A newer version of Chi is available!</a>
```
Comment (mattnickles) [inline src/website/assets/scripts/chi-docs.js]: Change text to `A new version of Chi is available! <a href="https://assets.ctl.io/chi/">Learn more</a>`

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/292/ for more details.

**Pull request merged**